lego/apps/surveys/tests/test_surveys_task.py

```diff
@@ -0,0 +1,25 @@
+from django.utils import timezone
+
+from lego.apps.surveys.models import Survey
+from lego.apps.surveys.tasks import send_survey_mail
+from lego.utils.test_utils import BaseTestCase
+
+
+class SurveyMailTestCase(BaseTestCase):
+    fixtures = [
+        'test_abakus_groups.yaml', 'test_users.yaml', 'test_companies.yaml', 'test_events.yaml',
+        'test_surveys.yaml'
+    ]
+
+    def setUp(self):
+        self.surveys = Survey.objects.all()
+
+    def test_sent_true(self):
+        active_surveys = self.surveys.filter(active_from__lte=timezone.now())
+        for survey in active_surveys:
+            self.assertEqual(survey.sent, False)
+
+        send_survey_mail()
+        sent_surveys = self.surveys.filter(active_from__lte=timezone.now())
```

Comment:
instead of doing the filtering twice you can just loop over `active_surveys` again and call `survey.refresh_from_db()` before the assert (that way you're sure that you're checking the same items as well).

Should maybe add an assert to make sure that `active_surveys` contain X amount of items as well, so it's not empty?